src/util.ts

```diff
@@ -1,18 +1,18 @@
 export function objectMap<T, R>(obj: Record<string, T>, func: (key: string, value: T) => R): R[] {
-    const result = []
-    for(const key of Object.keys(obj).sort()) {
-        result.push(func(key, obj[key]))
-    }
-    return result
+  const result = []
+  for(const key of Object.keys(obj).sort()) {
+    result.push(func(key, obj[key]))
+  }
+  return result
 }
 
-export function isMd() {
-    return matchMedia("(min-width: 768px)").matches
+export function isMobileDevice() {
```

Comment:
it was called it `isMd` because it's equal to tailwind's `md:`

Reply:
yeah but is mobile device is more readable

Reply:
fair

Reply:
md is bigger than mobile iirc?

Reply:
yeah md: is medium screens and above so mobileDevice isnt correct, either change name or invert the return

Reply:
Make your own pull request